### ISSUE TITLE
[BUG] Upgrade to 17 from 16 using webpack doesn't work

### ISSUE THREAD

**Issue by kicaj** [tabler/tabler#1456]
Configuration based on webpack.

**package.js:**
```
{
  "name": "test",
  "dependencies": {
    "@tabler/core": "^1.0.0-beta17"
  },
  "devDependencies": {
    "css-loader": "^6.7.3",
    "node-sass": "^8.0.0",
    "postcss-loader": "^7.0.2",
    "sass-loader": "^13.2.0",
    "style-loader": "^3.3.1",
    "webpack": "^5.75.0",
    "webpack-cli": "^5.0.1"
  }
}
```

**webpack.config.js:**
```
const path = require('path');

module.exports = {
  entry: './index.js',
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, './dist'),
  },
  module: {
    rules: [
      {
        test: /\.(scss)$/,
        use: [
          'style-loader',
          'css-loader',
          {
            loader: 'postcss-loader',
            options: {
              postcssOptions: {
                plugins: function () {
                  return [
                    require('autoprefixer')
                  ];
                }
              },
            },
          },
          'sass-loader',
        ],
      },
    ],
  },
};
```
**index.js:**
```
import './index.scss';
import '@tabler/core';
```
**index.scss:**
```
// Variables
$assets-base: '~@tabler/core/dist';

// Imports
@import '@tabler/core/src/scss/tabler';
```

Everything works with Tabler version 1.0.0-beta15 and older.

**Docker run with:**
 - node v18.14.0
 - npm 9.3.1
 - webpack 5.75.0
 - webpack-cli 5.0.1

**To reproduce**

1. Install packages
2. Create 4 files like above
3. Run command `npx webpack`

**Problems**
```
SassError: File to import not found or unreadable: ../../node_modules/bootstrap/scss/functions.
        on line 2 of node_modules/@tabler/core/src/scss/_bootstrap-config.scss
        from line 7 of node_modules/@tabler/core/src/scss/_config.scss
        from line 1 of node_modules/@tabler/core/src/scss/_core.scss
        from line 1 of node_modules/@tabler/core/src/scss/tabler.scss
        from line 5 of index.scss
>> @import "../../node_modules/bootstrap/scss/functions";
```

**Fixes:**
Replace path to bootstrap scss files by "bootstrap/..." or "~bootstrap/..." instead of "../../bootstrap/...".

**Comment by kicaj** [comment]
PR: https://github.com/tabler/tabler/pull/1457

**Comment by kicaj** [comment]
FYI: https://github.com/tabler/tabler/pull/1384/commits/d29e5f1b46e7cad81c9f8eaed88c15dea9f5c868

**Comment by kicaj** [comment]
@codecalm What was the idea of changing the path to `../../node_modules/`?

**Comment by SyntaxJoker** [comment]
We will be fixing that in https://github.com/tabler/tabler/pull/1477

**Comment by kevinpapst** [comment]
Fixed with #1477

You can try `dev` branch and let us know if it does not work for you @kicaj

**Comment by kicaj** [comment]
I confirm, thanks!
